firebase-firestore/src/androidMain/kotlin/dev/gitlive/firebase/firestore/FirebaseDocumentReferenceSerializer.kt

```diff
@@ -0,0 +1,5 @@
+package dev.gitlive.firebase.firestore
+
+actual val DocumentReference.platformValue: Any get() = android
```

Comment:
the kdoc is added at `expect` declaration shall it also be duplicated here?

Reply:
Do you know what the generated kdoc looks like for expect/actual ?